modules/zivid/camera.py

```diff
@@ -48,9 +48,128 @@ def __str__(self):
+        camera. If you invoke `capture_2d_3d` or `capture_3d` when there are two uncompleted 3D captures in-progress, then
+        the capture will not start until the first of the in-progress 3D captures has finished all transfer and
+        processing. There is a similar limit of maximum two in-process 2D captures per camera.
+
+        Capture functions can also be called on multiple cameras simultaneously. However, if the cameras have
+        overlapping field-of-view then you need to take consideration and sequence the capture calls to avoid the
+        captures interfering with each other.
+
+        Args:
+            settings: Settings to use for the capture.
+
+        Returns:
+            A frame containing a 3D point cloud, a 2D color image, and metadata.
+
+        Raises:
+            TypeError: If the settings argument is not a Settings instance.
+        """
+        if not isinstance(settings, Settings):
+            raise TypeError(
+                "Unsupported type for argument settings. Got {}, expected {}.".format(
+                    type(settings), Settings.__name__
+                )
+            )
+        return Frame(self.__impl.capture_2d_3d(_to_internal_settings(settings)))
+
+    def capture_3d(self, settings):
+        """Capture a single 3D frame.
+
+        This method is used to capture a 3D frame without a 2D color image. It ignores all color settings in the input
+        settings. See `capture_2d_3d` for capturing a 2D+3D frame.
+
+        This method returns right after the acquisition of the images is complete, and the camera has stopped projecting
+        patterns. For more information, see the remarks section of `capture_2d_3d` above. Those remarks apply for both 2D,
+        3D, and 2D+3D captures.
+
+        Args:
+            settings: Settings to use for the capture.
+
+        Returns:
+            A frame containing a 3D point cloud and metadata.
+
+        Raises:
+            TypeError: If the settings argument is not a Settings
+        """
+        if not isinstance(settings, Settings):
+            raise TypeError(
+                "Unsupported type for argument settings. Got {}, expected {}.".format(
+                    type(settings), Settings.__name__
+                )
+            )
+        return Frame(self.__impl.capture_3d(_to_internal_settings(settings)))
+
+    def capture_2d(self, settings):
+        """Capture a single 2D frame.
+
+        This method returns right after the acquisition of the images is complete, and the camera has stopped projecting
+        patterns. For more information, see the remarks section of `capture_2d_3d` above. Those remarks apply for both 2D,
+        3D, and 2D+3D captures.
+
+        This overload is provided for convenience. Note that only the Settings2D part under `Settings.color` will be
+        used for the capture. The other parts of the settings will be ignored.
+
+        This method will throw if `Settings.color` is not set.
```

Comment:
This method is used for capture_2d(settings) and capture_2d(settings_2d) it seems, this part of the docstring applies only for the former and not the latter. This could be clarified a bit in the docstring?

Reply:
Fixed